components/03-forms/10-misc/list-button/list-button.hbs

```diff
@@ -0,0 +1,5 @@
+<button type="{{ type }}" class="btn btn-list-button{{#if modifier }} list-button-{{ modifier }}{{/if}}" {{{ifattr 'disabled' disabled }}}>
+  <span>{{ label }}</span>
+  {{#if icon }}{{> '@icon' icon='check' }}{{/if}}
```

Comment:
In dit specifieke geval heeft het icoon meer dan alleen een decoratieve functie, het geeft ook aan of het item daadwerkelijk geselecteerd is. Dat zou dan op zich prima tekstueel aangeduid kunnen worden (visually hidden, als het vanuit het ontwerp het NIET de bedoeling is dat dit explciet wordt).

Reply:
Ik heb een `<span class="sr-only">geselecteerd</span>` toegevoegd.